.gitignore

```diff
@@ -11,9 +11,11 @@ rest-client.private.env.json
 .idea/workspace.xml
 .idea/shelf
 
-#Composer
+# Composer
 vendor
 composer.lock
 
 .php_cs.cache
 .phpunit.result.cache
+
+ReflectionData.json
```

Comment:
I just saw that the shell script also tries to remove this file in the end, but maybe gitignore'ing it too would be safer?

Reply:
Agree